crates/tree-sitter-qmd/tree-sitter-markdown/src/scanner.c

```diff
@@ -831,7 +830,7 @@ static bool parse_ordered_list_marker(Scanner *s, TSLexer *lexer,
         size_t digits = 1;
         bool dont_interrupt = lexer->lookahead != '1';
         advance(s, lexer);
-        while (isdigit(lexer->lookahead)) {
+        while (iswdigit(lexer->lookahead)) {
```

Comment:
Using tree-sitter blessed function https://github.com/tree-sitter/tree-sitter/blob/0be215e152d58351d2691484b4398ceff041f2fb/lib/src/wasm/stdlib-symbols.txt#L6

Which the shims also provided